test/functional/specs/Personalization/C782718.js

```diff
@@ -41,33 +41,28 @@ const getDecisionsMetaByScope = (decisions, scope) => {
     if (decision.scope === scope) {
       metas.push({
         id: decision.id,
-        scope: decision.scope
+        scope: decision.scope,
+        scopeDetails: decision.scopeDetails
       });
     }
   });
   return metas;
 };
 
-// This test is skipped until the related activities are properly configured.
-// https://jira.corp.adobe.com/browse/PDCL-4597
-test.skip("Test C782718: SPA support with auto-rendering and view notifications", async () => {
-  const alloy = createAlloyProxy();
-  await alloy.configure(config);
-
-  await alloy.sendEvent({
+const simulatePageLoad = async alloy => {
+  const resultingObject = await alloy.sendEvent({
     renderDecisions: true,
-    decisionScopes: [],
     xdm: {
       web: {
         webPageDetails: {
-          viewName: "/products"
+          viewName: "products"
         }
       }
     }
   });
-  await responseStatus(networkLogger.edgeEndpointLogs.requests, 200);
 
-  await t.expect(networkLogger.edgeEndpointLogs.requests.length).eql(3);
+  // asserts the request fired to Experience Edge has the expected event query
+  await responseStatus(networkLogger.edgeEndpointLogs.requests, 200);
```

Comment:
@ninaceban, as you continue work on tests in the future, I'd love it if you could help me out with this: https://github.com/adobe/alloy/pull/737#issuecomment-854817123 I'm hoping we can slowly (or quickly!) improve these tests over time to be more customer-oriented and less implementation-details-oriented. No need to change things for this PR though.

Reply:
Yes, let's discuss about it.